runtime/cel/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2025 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// cel provides utilities for evaluating Common Expression Language (CEL) expressions.
```

Comment:
```suggestion
// cel extends kstatus with custom readers for evaluating the health of 
// Kubernetes custom resources using Common Expression Language (CEL) expressions.
```

Reply:
Back to the original description. We should use this package for other CEL use cases across Flux as well, e.g. for the Receiver use case, which also needs to evaluate boolean expressions. I have modified this PR to introduce support for the Receiver CEL PR.